### PR TITLE
2to3: add French translation

### PR DESCRIPTION
**Pull request by Nico385412**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/2to3.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> Conversion automatisé de code python 2 vers 3.
+> Plus d'informations: <https://docs.python.org/3/library/2to3.html>.
```
Comment (navarroaxel) [inline pages.fr/common/2to3.md:4]: We use this French grammar rule:
```suggestion
> Plus d'informations : <https://docs.python.org/3/library/2to3.html>.
```

File: pages.fr/common/2to3.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# 2to3
+
+> Conversion automatisé de code python 2 vers 3.
+> Plus d'informations : <https://docs.python.org/3/library/2to3.html>.
+
+- Afficher les changements qui seront effectués sans les effectuer (coup d'essai):
```
Comment (navarroaxel) [inline pages.fr/common/2to3.md:6]: Here too
```suggestion
- Afficher les changements qui seront effectués sans les effectuer (coup d'essai) :
```

File: pages.fr/common/2to3.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+`2to3 {{chemin/vers/fichier.py}}`
+
+- Convertir un fichier Python 2 en Python 3:
```
Comment (navarroaxel) [inline pages.fr/common/2to3.md:10]:
```suggestion
- Convertir un fichier Python 2 en Python 3 :
```

File: pages.fr/common/2to3.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+- Convertir des fonctionnalités spécifiques de Python 2 vers Python 3:
+
+`2to3 --write {{chemin/vers/fichier.py}} --fix={{raw_input}} --fix={{print}}`
+
+- Convertir toutes les fonctionnalités de Python 2 vers Python 3 sauf exceptions spécifiques:
+
+`2to3 --write {{chemin/vers/fichier.py}} --nofix={{has_key}} --nofix={{isinstance}}`
+
+- Afficher une liste de toutes les fonctionnalités de language qui peuvent être converties de Python 2 vers Python 3:
```
Comment (navarroaxel) [inline pages.fr/common/2to3.md:22]:
```suggestion
- Afficher une liste de toutes les fonctionnalités de language qui peuvent être converties de Python 2 vers Python 3 :
```

File: pages.fr/common/2to3.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+- Convertir un fichier Python 2 en Python 3:
+
+`2to3 --write {{chemin/vers/fichier.py}}`
+
+- Convertir des fonctionnalités spécifiques de Python 2 vers Python 3:
+
+`2to3 --write {{chemin/vers/fichier.py}} --fix={{raw_input}} --fix={{print}}`
+
+- Convertir toutes les fonctionnalités de Python 2 vers Python 3 sauf exceptions spécifiques:
```
Comment (navarroaxel) [inline pages.fr/common/2to3.md:18]:
```suggestion
- Convertir toutes les fonctionnalités de Python 2 vers Python 3 sauf exceptions spécifiques :
```

File: pages.fr/common/2to3.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Afficher les changements qui seront effectués sans les effectuer (coup d'essai):
+
+`2to3 {{chemin/vers/fichier.py}}`
+
+- Convertir un fichier Python 2 en Python 3:
+
+`2to3 --write {{chemin/vers/fichier.py}}`
+
+- Convertir des fonctionnalités spécifiques de Python 2 vers Python 3:
```
Comment (navarroaxel) [inline pages.fr/common/2to3.md:14]:
```suggestion
- Convertir des fonctionnalités spécifiques de Python 2 vers Python 3 :
```

File: pages.fr/common/2to3.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+- Convertir toutes les fonctionnalités de Python 2 vers Python 3 sauf exceptions spécifiques:
+
+`2to3 --write {{chemin/vers/fichier.py}} --nofix={{has_key}} --nofix={{isinstance}}`
+
+- Afficher une liste de toutes les fonctionnalités de language qui peuvent être converties de Python 2 vers Python 3:
+
+`2to3 --list-fixes`
+
+- Convertir tous les fichier Python2 vers Python 3 dans un dossier:
```
Comment (navarroaxel) [inline pages.fr/common/2to3.md:26]:
```suggestion
- Convertir tous les fichier Python2 vers Python 3 dans un dossier :
```

File: pages.fr/common/2to3.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+`2to3 --output-dir={{chemin/vers/dossier_python3}} --write-unchanged-files --nobackups {{chemin/vers/dossier_python2}}`
+
+- Executer 2to3 avec plusieurs fil d'exécution:
```
Comment (navarroaxel) [inline pages.fr/common/2to3.md:30]:
```suggestion
- Executer 2to3 avec plusieurs fil d'exécution :
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by navarroaxel — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**